README.md

```diff
@@ -1,3 +1,20 @@
 # AlgebraicStockFlow
 [![Tests](https://github.com/AlgebraicJulia/StockFlow.jl/actions/workflows/tests.yml/badge.svg)](https://github.com/AlgebraicJulia/StockFlow.jl/actions/workflows/tests.yml)
 <!-- TODO: Set up on codecov.io for repo [![codecov](https://codecov.io/github/AlgebraicJulia/StockFlow.jl/branch/maaster/graph/badge.svg)](https://app.codecov.io/github/AlgebraicJulia/StockFlow.jl) -->
+
+StockFlow.jl is a Julia library for creating stock and flow diagrams using category theory.  Stockflows are used to represent systems where its population can move between different states, such as tracking how many people have been infected or recovered from a disease.  By providing initial values, and values for the parameters, we can solve the differential equation which the stockflow represents, which gives us a graph for how the population varies over time.
```

Comment:
Would it also be a good idea to phrase this using more common terminology e.g. instead of 'Stockflows' maybe 'Stock and flow diagrams' or, from the paper, 'stock-flow diagrams'?

Reply:
Yeah, I just used 'Stockflow' for brevity.

Reply:
Maybe we should mention causal loop diagrams as well?